requirements.txt

```diff
@@ -28,7 +28,7 @@ psycopg2-binary==2.8.6
 PyJWT==2.1.0
 pytz==2021.1
 PyYAML==5.4.1
-SQLAlchemy==1.3.23
+SQLAlchemy==1.4.37
```

Comment:
Well I guess it's good to bump it up anyway, I am surprised to see it upgraded though, does SQLAlchemy require a certain version of pytest? 🤔

Reply:
yup, old version of sqlalchemy doesnt work with newer version of pytest.